src/System.Globalization.Extensions/src/ApiCompatBaseline.net463.txt

```diff
@@ -0,0 +1,2 @@
+TypesMustExist : Type 'System.StringNormalizationExtensions' does not exist in the implementation but it does exist in the contract.
```

Comment:
Can you please add a comment similar to https://github.com/dotnet/corefx/blob/master/src/System.Security.SecureString/src/ApiCompatBaseline.txt#L2.